test/e2e/multihoming.go

```diff
@@ -637,11 +691,14 @@ func netCIDR(netCIDR string, netPrefixLengthPerNode int) string {
 	return fmt.Sprintf("%s/%d", netCIDR, netPrefixLengthPerNode)
 }
 
-func popCIDRsPerNodePrefix(netCIDR string, netPrefixLengthPerNode int) string {
-	if strings.Count(netCIDR, "/") > 1 {
-		return strings.ReplaceAll(netCIDR, fmt.Sprintf("/%d", netPrefixLengthPerNode), "")
+func getNetCIDRSubnet(netCIDR string) (string, error) {
```

Comment:
I will not block over this, but I don't understand why you're changing the signature. 

Would you explain ?

Reply:
because in your old code, it passes the netPrefixLengthPerNode argument as 24, that is not right for ipv6 subnet, which should be 64. I am just making this function more general that can be called for both types of networks.